Starfire's loadouts for the Tornado GR4

MetalStormGhost opened the pull request:
A modified version of Starfire's DCS World 2.7.9.17830 open beta loadouts for the Tornado GR4.

SEAD: 4xALARM, ECM pod, BOZ-107 countermeasure pod
DEAD: 2xALARM, 3xGBU-12, TGP, ECM pod, BOZ-107 countermeasure pod
Strike: TGP, 3xGBU-12, 2xAIM-9M, ECM pod, BOZ-107 countermeasure pod
CAS/BAI/OCA Aircraft: 4xBL-755 cluster bombs, 2xAIM-9M, ECM pod, BOZ-107 countermeasure pod
Anti-ship: 2xSea Eagle ASM, 2xAIM-9M, ECM pod, BOZ-107 countermeasure pod

Intended to address #1960

MetalStormGhost committed:
Removed SEAD capability from the Tornado GR4 as suggested by @SnappyComebacks (Resolves #1960). The Tornado GR4 still retains its DEAD mission capability.

MetalStormGhost committed:
Removed SEAD capability from the Tornado GR4 as suggested by @SnappyComebacks (Resolves #1960). The Tornado GR4 still retains its DEAD mission capability.

MetalStormGhost committed:
Updated the changelog.

RndName commented on changelog.md:
```diff
@@ -12,6 +12,7 @@ Saves from 5.x are not compatible with 6.0.
 * **[Flight Planning]** Added the ability to plan tankers for recovery on package flights.  AI does not plan.
 * **[Flight Planning]** Air to Ground flights now have ECM enabled on lock at the join point, and SEAD/DEAD also have ECM enabled on detection and lock at ingress.
 * **[Flight Planning]** AWACS flightplan changed from orbit to a racetrack to reduce data link disconnects which were caused by blind spots as a result of the bank angle. 
+* **[Flight Planning]** The Tornado GR4 will no longer be tasked for SEAD missions due to no access to ARMs.
```
It's more a fix than a feature :) you can also feel free to squash the two commits to one. no need for a separate changelog commit.

MetalStormGhost replied:
Thanks for noticing that! It's now in the correct place.

MetalStormGhost committed:
Merge branch 'tornado-gr4-sead-capability' of https://github.com/MetalStormGhost/dcs_liberation into tornado-gr4-sead-capability

SnappyComebacks commented:
Should the DEAD tasking be kept?  I noticed there is no DEAD loadout (and it would also be identical to the CAS loadout)

SnappyComebacks commented:
@Hornet2041 mentioned that the GR4 carries the `ALARM` missile, which is a `ARM`.  So perhaps we should keep SEAD and use that instead.  One thing to note is that the `ALARM` missile sits in the same place as the four inner pylons.

![overlap_gr4](https://user-images.githubusercontent.com/74509817/150178096-b940373d-241a-4ef8-a22c-a4c16e3aebba.jpg)

MetalStormGhost committed:
Revert "Updated the changelog."

This reverts commit 7eb82519e063a6d3acf92576be1774ff30f9c49a.

MetalStormGhost committed:
Revert "Removed SEAD capability from the Tornado GR4 as suggested by @SnappyComebacks (Resolves #1960). The Tornado GR4 still retains its DEAD mission capability."

This reverts commit b5c269fe788de8bed4cd01ccbe0206b6aa4a343c.

MetalStormGhost committed:
Starfire's DCS World 2.7.9.17830 open beta loadout for the Tornado GR4. See #1870

SEAD: 4xALARM, ECM pod, BOZ-107 countermeasure pod
DEAD: 2xALARM, 2xGBU-16, ECM pod, BOZ-107 countermeasure pod
Strike: 4xGBU-12, 2xAIM-9M, ECM pod, BOZ-107 countermeasure pod
CAS/BAI/OCA Aircraft: 4xBL-755 cluster bombs, 2xAIM-9M, ECM pod, BOZ-107 countermeasure pod
Anti-ship: 2xSea Eagle ASM, 2xAIM-9M, ECM pod, BOZ-107 countermeasure pod

SnappyComebacks commented:
I ran this branch to look at it and found this:

The Liberation UI shows that these loadouts have no air to ground munitions:

- Liberation BAI
- Liberation CAS
- Liberation OCA/Aircraft

![image](https://user-images.githubusercontent.com/74509817/150282205-fce6d4ab-aeb0-4b91-a244-e42ca7580ecd.png)

But that is not true and I have no idea if it's Liberation's fault, PyDCS' fault, or DCS' fault, because when you generate the mission, there are munitions there, but they are the clusters that cannot be directly selected from the mission editor, and only selected from the mission editor loadouts.  I have a feeling the root cause may be DCS.


Second, are we ok with leaving off the TGP with AI laser guided bombs?  I know to AI doesn't need them, I just don't know if we have said in the past if we should do one way or the other.  (If you want to punt this that's ok.  It's just something I noticed here.)

DanAlbert commented:
> Second, are we ok with leaving off the TGP with AI laser guided bombs? I know to AI doesn't need them, I just don't know if we have said in the past if we should do one way or the other.

I've always leaned toward including things like that. If it makes for a more believable presentation and costs us the same amount of effort to do as to not do, why not?

DanAlbert reviewed (changes requested):
https://github.com/dcs-liberation/dcs_liberation/pull/1961#issuecomment-1017155327 appears unaddressed

MetalStormGhost committed:
Changed the Strike loadout to TGP, 3xGBU-12, 2xAIM-9M, ECM pod, BOZ-107 countermeasure pod.

MetalStormGhost committed:
Changed the DEAD loadout to 2xALARM, 3xGBU-12, TGP, ECM pod, BOZ-107 countermeasure pod.

RndName commented:
What is the status of this PR and what does it change? For me it is totaly unclear at what i look atm. I think it is not the initial description provided in the PR anymore? Can you update the description and Squash your changes?
Also is there any tester who can verify these changes?

MetalStormGhost commented:
> What is the status of this PR and what does it change? For me it is totaly unclear at what i look atm. I think it is not the initial description provided in the PR anymore? Can you update the description and Squash your changes? Also is there any tester who can verify these changes?

Description updated.

@SnappyComebacks could I ask you to test if this would fix #1960 ?

SnappyComebacks commented:
> @SnappyComebacks could I ask you to test if this would fix #1960 ?

I likely won't be able to.

I do have two follow up questions:

- What did you discover about the missing CBUs in the Liberation UI?

- Should the `Strike` be multiple GBU-12s, or one GBU-16?  The implication being that the type of Strike objectives that a Tornado can handle is decided by this.  I lean towards multiple GBU-12s, as I feel like buildings with high resistance would require a GBU-10, that the Tornado cannot use.

MetalStormGhost commented:
> I do have two follow up questions:
> 
>     * What did you discover about the missing CBUs in the Liberation UI?
> 
>     * Should the `Strike` be multiple GBU-12s, or one GBU-16?  The implication being that the type of Strike objectives that a Tornado can handle is decided by this.  I lean towards multiple GBU-12s, as I feel like buildings with high resistance would require a GBU-10, that the Tornado cannot use.

The CBUs are missing from the Liberation UI because they aren't a valid weapon for the Tornado (in the current open beta of DCS at least), as it turns out. You cannot create a loadout for either Tornado in the ME with the CBUs on the fuselage hardpoints.

I agree with the above comment regarding GBU-10. I chose to go with several GBU-12s for Strike because I figured the area effect on e.g. FARP tent camps would be better, and a single GBU-16 would not be able to destroy a hardened bunker anyway.
